content/en/docs/v3.5/op-guide/configuration.md

```diff
@@ -28,7 +28,7 @@ Flags are presented below using the format `--flag-name DEFAULT_VALUE`.
   **Note**: For details concerning new, updated, and deprecated {{< param version >}} flags,
   see [CHANGELOG-{{< psubstr version 1 >}}.md][changelog].
 
-  [changelog]: https://github.com/etcd-io/etcd/blob/main/CHANGELOG-{{< psubstr version 1 >}}.md
+  [changelog]: https://github.com/etcd-io/etcd/blob/main/CHANGELOG/CHANGELOG-{{< psubstr version 1 >}}.md
```

Comment:
We also need to fix this link in the 3.6 alpha docs.  If you have time @siansiansu please push a new commit to include that. 

There has been quite a long time since you raised this pr so I can pick this up once this is merged if necessary.